web/templates/pages/tasks.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+		      <h1>All CodeCatz Tasks</h1>
+		</div>
+	</header>
+
+	{% if task_list %}
+
+		<div class="row">
+
+			{% for task in task_list %}
+
+				<div class="col-md-3 col-xs-6">
+					<h3><strong>{{ task.title }}</strong></h3>
+					<p><strong>Assigned to:</strong> {{ task.assigned_id }}</p>
+
+					<p><strong>Project status:<strong> {{ task.get_status_display }} </p>
+					<p><strong>Part of project:</strong><a href="{% url 'web.views.project_detail' project_id=task.project_id.project_id %}"> {{ task.project_id.project_name }}</a></p>
```

Comment:
could you put the space (again, I know) outside the link tag, so it isn't a link as well?
so instead of
`<p><strong>Part of project:</strong><a href="{% url 'web.views.project_detail' project_id=task.project_id.project_id %}"> {{ task.project_id.project_name }}</a></p>`
do this
`<p><strong>Part of project:</strong> <a href="{% url 'web.views.project_detail' project_id=task.project_id.project_id %}">{{ task.project_id.project_name }}</a></p>`